projects/angular/src/layout/vertical-nav/STYLES.md

```diff
@@ -0,0 +1,37 @@
+# Vertical Nav
+
+## CSS Custom Properties
+
+| CSS Custom Property                             | Description                                |
+| ----------------------------------------------- | ------------------------------------------ |
+| --clr-vertical-nav-divider-color                | Vertical nav divider color.                |
```

Comment:
`clr-vertical-nav-divider-color` it's not used

Reply:
omitted in commit 95c7d81